README.md

```diff
@@ -1,47 +1,56 @@
 # Mantle: Gluing CoreOS together
 
-This repository is a collection of utilities for the CoreOS SDK.
-
-## plume
-
-CoreOS release utility
-
-## plume index
-
-Generate and upload index.html objects to turn a Google Cloud Storage
-bucket into a publicly browsable file tree. Useful if you want something
-like Apache's directory index for your software download repository.
+This repository is a collection of utilities for the Container Linux SDK. Most of the
```

Comment:
I think "for developing" sounds better.

Right now, the release process is the only place that uses `gangue` by default: https://github.com/coreos/scripts/blob/master/jenkins/images.sh#L14,L17

I've enabled it in my own SDK by writing the following to `/etc/env.d/99gangue` and running `env-update`:

```sh
COREOS_DEV_BUILDS='gs://builds.developer.core-os.net'
FETCHCOMMAND_GS='/usr/bin/gangue get --json-key=/dev/null "${URI}" "${DISTDIR}/${FILE}"'
RESUMECOMMAND_GS='/usr/bin/gangue get --json-key=/dev/null "${URI}" "${DISTDIR}/${FILE}"'
```

The `/dev/null` causes it to fall back to my user's authorization, but that path normally takes service account credentials.